### PR TITLE
Fix sonar qube smells

### PR DESCRIPTION
**Pull request by phillipDenness**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/coh/controller/decision/DecisionResponse.java [hunk 1]
```diff
@@ -5,10 +5,11 @@
 import uk.gov.hmcts.reform.coh.controller.decisionreplies.DecisionReplyResponse;
 import uk.gov.hmcts.reform.coh.controller.state.StateResponse;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
-public class DecisionResponse {
+public class DecisionResponse implements Serializable {
```
Comment (johnhosolirius) [inline src/main/java/uk/gov/hmcts/reform/coh/controller/decision/DecisionResponse.java]: Why Serializable?

File: src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java [hunk 2]
```diff
@@ -44,6 +44,8 @@
     private static final String STARTING_STATE = DecisionsStates.DECISION_DRAFTED.getStateName();
 
     private static final String PENDING_STATE = DecisionsStates.DECISION_ISSUE_PENDING.getStateName();
+    private static final String ONLINE_HEARING_NOT_FOUND = "Online hearing not found";
```
Comment (johnhosolirius) [inline src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java]: Let's move ONLINE_HEARING_NOT_FOUND somewhere common as it being used in so many places

File: src/main/java/uk/gov/hmcts/reform/coh/controller/decision/DecisionResponse.java (hunk 1 above)
Comment (phillipDenness) [reply src/main/java/uk/gov/hmcts/reform/coh/controller/decision/DecisionResponse.java]: Jackson wants serializable objects ? Also sonarqube underlined it in red and told me to make it serializable

Comment (johnhosolirius) [reply src/main/java/uk/gov/hmcts/reform/coh/controller/decision/DecisionResponse.java]: Hmm. Jackson already serialises it 🤷‍♂️ Okay. Just ignore this comment

**Pull request merged**